src/blocks/scratch3_control.js

```diff
@@ -35,7 +35,8 @@ class Scratch3ControlBlocks {
             control_delete_this_clone: this.deleteClone,
             control_get_counter: this.getCounter,
             control_incr_counter: this.incrCounter,
-            control_clear_counter: this.clearCounter
+            control_clear_counter: this.clearCounter,
```

Comment:
nit: are we caring about alphabetization at this point yet?

Reply:
I don't see exactly what you mean?

We're definitely sticking to `camelCase` for function/method names. I try to follow the same opcode-case that other opcodes in the same category use, so in this case, `snake_case`.

Reply:
Oh, got it. Arguably they should be in the order (get, clear, incr) because [that's the way they are in scratch-flash](https://github.com/LLK/scratch-flash/blob/35b6a84bfd508c17105d6bf68cd0962a5c8c94a0/src/Specs.as#L379-L381), or alphabetic, because consistency.. but the thing is, this PR adds "all at once", not the counter blocks (that was PR #1033). Maybe it's worth it to clean this up at some point, but the order definitely isn't causing any confusion right now.